transaction/std/src/transaction_builder.rs

```diff
@@ -23,6 +23,12 @@ use mc_transaction_core::{
 use mc_util_from_random::FromRandom;
 use rand_core::{CryptoRng, RngCore};
 
+/// This is a datatype for the comparer of the transaction outputs
+pub type TxComparerFn = fn(&TxOut, &TxOut) -> core::cmp::Ordering;
+
+/// This is a datatype helper used for transaction outputs sorting mechanism
+pub type TxOutputsComparerOption = Option<TxComparerFn>;
+
```

Comment:
When doing stuff like this you should just use the Option in-situ.

Reply:
not using anymore, fixed